### PR TITLE
Show current and default values in config dialog (pdf)

### PR DESCRIPTION
**Pull request by robert00s**
Similar to #3952 but for pdf.

### CODE REVIEW

**Comment by poire-z** [comment]
A bit bothering that the showValues code is duplicated (exactly the same?) in both creoptions.lua and koptoptions.lua. Could that be put into another file? ui/data/optionutils.lua (or better name)?

**Comment by robert00s** [comment]
@poire-z Done :)

**Comment by poire-z** [comment]
Thanks!
I think you could even avoid that kind of wrapper:
```
local showValues = function(configurable, option, prefix)
    OptionsUtil:showValues(configurable, option, prefix)
end
```
and just use `local showValues = OptionsUtil.showValues` (or use `OptionsUtil.showValues` directly in the config table) if you were using a dot `.` instead of a colon `:` for the OptionsUtil functions (make them plain functions instead of methods, as you don't use any `self` at all in them).
Like it's done for all the functions in frontend/util.lua. Guess that then, OptionsUtil should be lowercased.

File: frontend/ui/data/optionsutil.lua [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+]]
+
+local Device = require("device")
+local InfoMessage = require("ui/widget/infomessage")
+local UIManager = require("ui/uimanager")
+local _ = require("gettext")
+local Screen = Device.screen
+local T = require("ffi/util").template
+
+local optionutil = {}
```
Comment (Frenzie) [inline frontend/ui/data/optionsutil.lua]: While it doesn't really matter for the code itself, I can't help but notice that the file and the use say option*s*util and in this file it all says optionutil. optionsutil probably sounds better. ;-)

**Pull request merged**